src/app/hooks/useReCaptcha.ts

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+
+export const useReCaptcha = () => {
+  const [key, setKey] = useState('');
+  const [token, setToken] = useState('');
+
+  const handleLoaded = () => {
+    const windows: any = window;
+
+    windows.grecaptcha.ready((_: any) => {
+      windows.grecaptcha.execute(key, { action: 'REGISTER' }).then((token: any) => {
+        setToken(token);
+      });
+    });
+  };
+
+  useEffect(() => {
+    const fetchKeys = async () => {
+      const response = await fetch('http://ftx.us/api/recaptcha/key');
```

Comment:
WDYT about extracting the api url to a env variable and the route to a constant?

Reply:
Yes! I'll use the one that we already have. Regarding the route, my idea is to know if this approach is possible and if works in staging. Knowing that this works as expected, I'll refactor this, adding it as a standard service and integrate it in the same way as the other fetchs.